actions.go

```diff
@@ -141,6 +143,15 @@ func ReadActionsYaml(r io.Reader) (*Actions, error) {
 					return nil, errors.Errorf("value for schema key %q must be a string", key)
 				}
 				thisActionSchema[key] = typed
+			case "nonblocking":
```

Comment:
Notice that if key "nonblocking" is not found in the map it then defaults to false. Thus existing actions that do not specify the property `NonBlocking` will be counted as having `NonBlocking: false` which their existing behavior.